### DISCUSSION TITLE
Issue at remark-language-server was closed with vague explanation

### DISCUSSION THREAD

**Discussion by txtyash** [remarkjs/remark#1032] [General] [answered]
https://github.com/remarkjs/remark-language-server/issues/7

This was an issue I made regarding the incorrect formatting of tasklists.

I primarily use lunarvim and the only working lsp available for lunarvim is remark_ls and that is why I made that issue.

I don't think it needs more explanation(Task lists are incorrectly formatted)

**Comment by wooorm** [comment]
Please provide more information, as requested. See https://stackoverflow.com/help/how-to-ask for help how to ask questions.

**Comment by ChristianMurphy** [comment, marked as answer]
It was closed because you opened 3 vague issues and didn't respond to requests for context.

As of today, the answer from https://github.com/remarkjs/remark/discussions/998#discussioncomment-2910461 still holds.

If you've tried this and it didn't work please elaborate.

**Comment by txtyash** [comment]
@ChristianMurphy I'm sorry I never saw that comment. That comment was the expected answer. My problem is solved now.
